init.c

```diff
@@ -408,6 +422,10 @@ static void parse_options(int argc, char *argv[], char **config_path,
 
 #define SIZE_CMDLINE_BUF 1024
 
+// somehow this should for appengine really take the
+// argv of the process instead of the /proc/cmdline
+// in turn appengine should make a file cmdline and mount
+// it into the containers as containers use that often
```

Comment:
this was a my TODO comment? wnate to keep it in? if so we should use XXX: to mark it as a todo or remove it again...

Reply:
it was your TODO comment yeah. Removing it as we already support this using the --cmdline arg on the init binary, or using the -c arg on the appengine run script